db/migrate/20190225190111_convert_all_claims_to_use_english_pdf.rb

```diff
@@ -0,0 +1,13 @@
+class ConvertAllClaimsToUseEnglishPdf < ActiveRecord::Migration[5.2]
```

Comment:
Use this migration to set all old claims to use this template - then the next migration to stop the field becoming null ever again